src/tcp.ts

```diff
@@ -7,6 +7,19 @@ import { ByteTransformer, SocketLogTransformer } from './lib/transformers'
 export function createTcpProxy(config: Config, logger: Logger) {
     const listener = createServer()
 
+    const configureSocketKeepAlive = (socket: any, keepAliveInterval: number, name: string) => {
+        try {
```

Comment:
Please put the try/catch inside the if statement, not the if inside the try/catch